### PR TITLE
feature/add_http_proxy add proxy cli variable

### PR DESCRIPTION
**Pull request by oldbig**
### The subscription:
```json
{ 
"endpoint": "https://fcm.googleapis.com/fcm/send/d61c5u920dw:APA91bEmnw8utjDYCqSRplFMVCzQMg9e5XxpYajvh37mv2QIlISdasBFLbFca9ZZ4Uqcya0ck-SP84YJUEnWsVr3mwYfaDB7vGtsDQuEpfDdcIqOX_wrCRkBW2NDWRZ9qUz9hSgtI3sY", 
  "expirationTime": null, 
  "keys": { 
    "p256dh": "BL7ELU24fJTAlH5Kyl8N6BDCac8u8li_U5PIwG963MOvdYs9s7LSzj8x_7v7RFdLZ9Eap50PiiyF5K0TDAis7t0", 
    "auth": "juarI8x__VnHvsOgfeAPHg" 
  } 
}
```

### The command:
```bash
web-push send-notification  \
--endpoint=https://fcm.googleapis.com/fcm/send/d61c5u920dw:APA91bEmnw8utjDYCqSRplFMVCzQMg9e5XxpYajvh37mv2QIlISdasBFLbFca9ZZ4Uqcya0ck-SP84YJUEnWsVr3mwYfaDB7vGtsDQuEpfDdcIqOX_wrCRkBW2NDWRZ9qUz9hSgtI3sY \
--key=BL7ELU24fJTAlH5Kyl8N6BDCac8u8li_U5PIwG963MOvdYs9s7LSzj8x_7v7RFdLZ9Eap50PiiyF5K0TDAis7t0 \
--auth=juarI8x__VnHvsOgfeAPHg \
--vapid-subject=mailto:example@qq.com \
--vapid-pubkey=BGtkbcjrO12YMoDuq2sCQeHlu47uPx3SHTgFKZFYiBW8Qr0D9vgyZSZPdw6_4ZFEI9Snk1VEAj2qTYI1I1YxBXE \
--vapid-pvtkey=I0_d0vnesxbBSUmlDdOKibGo6vEXRO-Vu88QlSlm5j0 \
--payload=Hello \
--proxy=http://127.0.0.1:8889
```

### CODE REVIEW

**Comment by marco-c** [comment]
Could you update the "Command Line" section in the README to mention the new argument?

File: src/cli.js [hunk 1]
```diff
@@ -87,6 +88,10 @@ const sendNotification = args => {
     };
   }
 
+  if (args['proxy']) {
```
Comment (marco-c) [inline src/cli.js:91]: There's a linting error to fix here

**Comment by oldbig** [comment]
> Could you update the "Command Line" section in the README to mention the new argument?

Ok, Fixed.

File: src/cli.js (hunk 1 above)
Comment (oldbig) [reply src/cli.js]: Fixed.

File: README.md [hunk 2]
```diff
@@ -77,6 +77,34 @@ Then you can run the following commands:
```
Comment (marco-c) [inline README.md:77]: At line 76, there is a full list of arguments, could you add the new proxy argument too?

File: README.md [hunk 3]
```diff
@@ -77,6 +77,34 @@ Then you can run the following commands:
...
+  ```shell
+    web-push send-notification  \
+    --endpoint=https://fcm.googleapis.com/fcm/send/d61c5u920dw:APA91bEmnw8utjDYCqSRplFMVCzQMg9e5XxpYajvh37mv2QIlISdasBFLbFca9ZZ4Uqcya0ck-SP84YJUEnWsVr3mwYfaDB7vGtsDQuEpfDdcIqOX_wrCRkBW2NDWRZ9qUz9hSgtI3sY \
+    --key=BL7ELU24fJTAlH5Kyl8N6BDCac8u8li_U5PIwG963MOvdYs9s7LSzj8x_7v7RFdLZ9Eap50PiiyF5K0TDAis7t0 \
+    --auth=juarI8x__VnHvsOgfeAPHg \
+    --vapid-subject=mailto:example@qq.com \
+    --vapid-pubkey=BGtkbcjrO12YMoDuq2sCQeHlu47uPx3SHTgFKZFYiBW8Qr0D9vgyZSZPdw6_4ZFEI9Snk1VEAj2qTYI1I1YxBXE \
+    --vapid-pvtkey=I0_d0vnesxbBSUmlDdOKibGo6vEXRO-Vu88QlSlm5j0 \
+    --payload=Hello \
+    --proxy=http://127.0.0.1:8889
```
Comment (marco-c) [inline README.md:106]: Let's remove `--proxy` from the example, as the example should be generic and `proxy` is not a super common option.

Comment (oldbig) [reply README.md]: Done.

**Review by marco-c — APPROVED** [review]
Thanks!

**Comment by marco-c** [comment]
I've released v3.5.0 with your fix.

**Pull request merged**